drivers/docker/driver.go

```diff
@@ -1107,14 +1107,30 @@ func (d *Driver) DestroyTask(taskID string, force bool) error {
 
 	c, err := h.client.InspectContainer(h.containerID)
 	if err != nil {
-		return fmt.Errorf("failed to inspect container state: %v", err)
-	}
-	if c.State.Running && !force {
-		return fmt.Errorf("must call StopTask for the given task before Destroy or set force to true")
-	}
+		switch err.(type) {
+		case *docker.NoSuchContainer:
+			h.logger.Error("failed to inspect container state, will proceed with DestroyTask",
```

Comment:
This feels a bit odd to be Error log considering that we handle it quite well.  I would downgrade it to info or debug with it indicating "container was deleted out-of-band, will proceed with DestroyTask"`.